lib/ddtrace/contrib/sinatra/tracer.rb

```diff
@@ -32,18 +36,19 @@ module Tracer
         option :tracer, default: Datadog.tracer
         option :resource_script_names, default: false
         option :distributed_tracing, default: false
+        option :headers, default: Utils::MassTagger::DEFAULT_HEADERS
```

Comment:
I like that both integrations will have the same set of default options. DRY is good!

However, something about the name of the constant it sources from doesn't sit quite right: why does a `MassTagger` own a list of default headers? Does tagging implicitly include HTTP? Or does HTTP instead implement tagging?

Reply:
I've moved that to RequestSpanMiddleware which I think might be more appropriate